wiki/jee.md

```diff
@@ -361,6 +363,102 @@ In order to do this, you must provide the following content in the
 
 ... here, the `<class-loader delegate="false" />` part is the relevant bit.
 
+### Deactivate MoxyJson
+
+Apparently GF4 ships with MoxyJson which may cause problems like this, when you try to parse JSON within your application.
+
+    [2016-09-01T16:27:37.782+0200] [Payara 4.1] [SEVERE] []   [org.glassfish.jersey.message.internal.WriterInterceptorExecutor] [tid: _ThreadID=28 _ThreadName=http-listener-1(3)] [timeMillis: 1472740057782] [levelValue: 1000] [[MessageBodyWriter not found for media type=application/json, type=class xxx.JsonClass, genericType=class xxx.JsonClass.]]
+
+There are two possible solutions available.
+
+Preferred solution:
+
+To integrate your own dependency you need to disable the MoxyJson with the below code.
+
+    @ApplicationPath("/")
+    public class ApplicationConfig extends Application {
+    
+      /**
+       * {@inheritDoc}
+       */
+      @Override
+      public Map<String, Object> getProperties() {
+        final Map<String, Object> properties = new HashMap<String, Object>();
+        properties.put("jersey.config.server.disableMoxyJson", true);
+    
+        return properties;
+      }
+    }
+
+Then add your own dependencies, e.g. in this case only those two because the others are referenced by the joynr lib itself.
+
+    <dependency>
+      <groupId>com.fasterxml.jackson.jaxrs</groupId>
+      <artifactId>jackson-jaxrs-json-provider</artifactId>
+      <version>2.6.2</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.dataformat</groupId>
+      <artifactId>jackson-dataformat-xml</artifactId>
+      <version>2.6.2</version>
+    </dependency>
+
+Finally in case you're using JSON: Not setting a value to the @JsonProperty annotations will cause a No MessageBodyWriter found exception. To avoid that use the following on relevant getters of your class.
+
+    @JsonProperty("randomName")
+    public String getRandomName(){
+    ...
+    }
+
+
+Alternative:
```

Comment:
Fourth level header 'Alternative solution'

I'm leaning towards not including this alternative solution, as it's a bit kludgy and if the solution above works, then that should be sufficient.